### ISSUE TITLE
ImportError: /content/faster-rcnn.pytorch/lib/model/roi_crop/_ext/roi_crop/_roi_crop.so: undefined symbol: _cudaRegisterFatBinaryEnd

### ISSUE THREAD

**Issue by JiaLim98** [jwyang/faster-rcnn.pytorch#824]
Hi,

Has anyone use this repo successfully using PyTorch 0.4.0 with torchvision 0.2.0 in Google Colab? I am facing this issue:
![image](https://user-images.githubusercontent.com/66768413/89709015-8d840d80-d9ae-11ea-9084-95f5b9ba3be0.png)
I have also posted [here](https://github.com/jwyang/faster-rcnn.pytorch/issues/724), awaiting for any responses. I recompiled all CUDA related libraries and have also tried [this](https://github.com/open-mmlab/mmdetection/issues/385), but no luck.
Any help would be much appreciated! Thanks in advance.

Best Regards.
JiaLim98

**Comment by SkeletonOne** [comment]
@JiaLim98 Any update about this issue?

**Comment by JiaLim98** [comment]
@SkeletonOne I don't how to overcome this issue, but I have get it to work on Pytorch 1.1.0. I completed a Colab notebook tutorial, only I set it up on my server following the commands I used in Colab. Perhaps you can try set it up on Colab first?

**Comment by JiaLim98** [comment]
@SkeletonOne I have not overcome this issue. I am still struggling to get this repo working. If you can get it to work, please let me know.

**Comment by Jiaxin-lucky** [comment]
I have the same problem, do you overcome it yet?

**Comment by hanhao2019102935** [comment]
I have the same problem, do you overcome it yet?

**Comment by hanhao2019102935** [comment]
@JiaLim98 Have you solved the problem?I had the same problem

**Comment by JiaLim98** [comment]
@hanhao2019102935 No, I did not manage to solve it. If you find any solution in the future, feel free to post here. I think a lot of people are facing this issue.

**Comment by hanhao2019102935** [comment]
@JiaLim98 I tried to run the project on coLab.It failed, I think, because the default version of Python in the coLab is 3.7, but the 3.7 version of Python is no longer able to install a torch below 0.4.0.So it didn't compile successfully.Do you have any other ideas?You can contact me VX: H178337111
